scripts/ci/pre_commit/check_min_python_version.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from __future__ import annotations
+
+import subprocess
+import sys
+from pathlib import Path
+
+sys.path.insert(0, str(Path(__file__).parent.resolve()))
+
+from common_precommit_utils import console
+
+required_version = (3, 9)
```

Comment:
As when we upgrade Python next time - can you make the version a bit more "catchy" such that if we search for "3.9" that we find this literal? :-D

Reply:
+1 even a comment would do

Reply:
Updated it - but I am not really sure if it's better (and adding 3.9 in the comment would be quite a duplication - but i added a generic comment about the upgrade):

```python
required_version = tuple(map(int, "3.9".split(".")))
```

Reply:
THANKS!

Reply:
You are welcome!